### PR TITLE
Political Factions Game Rule

### PR DESCRIPTION
**Pull request by zumbak04**
### Changelog:
- Added the Political Factions game rule that regulates the size and existence of the political factions. Has 4 options:
  - Medium (The AI doesn't join factions that have reached the limit of 200 Realm Size and 10 Faction Tributaries)
  - Large (The AI doesn't join factions that have reached the limit of 300 Realm Size and 15 Faction Tributaries)
  - Off (cannot be created)
  - Small (The AI doesn't join factions that have reached the limit of 100 Realm Size and 5 Faction Tributaries)

**_P.S. The AI can't join on its own, but you still can invite it._**

### How to Test
- Try to create the Alliance as Terenas or the Horde as Blackhand in the first bookmark. Try to play Terenas or Thrall in the second bookmark. See if the rule works.
- Try to disable the factions and fire the Exodus of the Horde event by using `event WCTHW.120`, your character should have the Horde title, but not the Leader of the Horde trait.
- Try to disable the factions and look at the Alliance in the second bookmark. It should be destroyed after Arthas surrenders to the Lich King.

### CODE REVIEW

**Comment by zumbak04** [comment]
@DukeJeans proofread please.

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2988,15 +2981,15 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS;�RWarcraft�!: Political Factions;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS_MEDIUM;Medium;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS_MEDIUM_DESC;The AI doesn't join the factions reached the limit of 200 Realm Size and 10 Faction Tributaries.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
RULE_POLITICAL_FACTIONS_MEDIUM_DESC;The AI doesn't join factions that have reached the limit of 200 Realm Size and 10 Faction Tributaries.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2988,15 +2981,15 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
...
+RULE_POLITICAL_FACTIONS_LARGE;Large;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS_LARGE_DESC;The AI doesn't join the factions reached the limit of 300 Realm Size and 15 Faction Tributaries.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
RULE_POLITICAL_FACTIONS_LARGE_DESC;The AI doesn't join factions that have reached the limit of 300 Realm Size and 15 Faction Tributaries.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -2988,15 +2981,15 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
...
+RULE_POLITICAL_FACTIONS_DISABLED;Off;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS_DISABLED_DESC;The political factions like the Alliance and the Horde cannot be created. The AI doesn't join the historical factions reached the limit of 100 Realm Size and 5 Faction Tributaries.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
RULE_POLITICAL_FACTIONS_DISABLED_DESC;Political factions like the Alliance and the Horde cannot be created. The AI doesn't join the historical factions that have reached the limit of 100 Realm Size and 5 Faction Tributaries.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 4]
```diff
@@ -2988,15 +2981,15 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
...
+RULE_POLITICAL_FACTIONS_SMALL;Small;;;;;;;;;;;;;x
+RULE_POLITICAL_FACTIONS_SMALL_DESC;The AI doesn't join the factions reached the limit of 100 Realm Size and 5 Faction Tributaries.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
RULE_POLITICAL_FACTIONS_SMALL_DESC;The AI doesn't join factions that have reached the limit of 100 Realm Size and 5 Faction Tributaries.;;;;;;;;;;;;;x
```

**Comment by BronzeBaron** [comment]
Works as intended, according to my own testing.

Set Political Factions to off, and;
- As expected; Used the ' 'Executed the exodus of the horde event, verified that I did not have the 'Leader of the Horde' trait. Ran it for a couple years past that, nothing out of the ordinary.
- As expected; Alliance was after Arthas surrenders to the Lich King, could not be reformed by alliance nations. Leadership trait gone from Arthas. 
- As expected; While I grew closer to the Realm Size limit from each rule, I did receive less requests to join the faction. Still able to manually invite to each faction though. (Tested this as both Horde and Alliance.)

**Comment by DukeJeans** [comment]
Fix your conflicts if you know what's good for you 🔫

**Pull request merged**